index.js

```diff
@@ -102,11 +103,25 @@ const init = (seed, option) => {
                 }));
             }));
         }));
-
         r.net.Connection.prototype.DEFAULT_PORT = port;
         t.context.dbPort = port;
     };
 };
+const restore = async (option) => {
+    const config = joi.attempt({ ...option }, joi.object().required().keys({
+        port     : joi.number().required(),
+        dump     : joi.string().required(),
+        password : joi.string().required()
+    }));
+
+    const filepath = await tempWrite(config.password);
+    const args = ['restore', option.dump, '--password-file', filepath, '--connect', `localhost:${config.port}`];
```

Comment:
`option.dump` -> `config.dump`

Also, we should probably make the password and port optional. Not everyone would want to use them.

See, for example, how we do it in `init()`.

```js
if (password) {
    args.push('--initial-password', password);
}
```

Reply:
I think `rethinkdb` always set `password` even if you don't tell it

Reply:
And for port it will work if you won't provide any `port` by deafault it will look for `28015`?